src/main/java/ch/njol/skript/expressions/ExprRawString.java

```diff
@@ -0,0 +1,100 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionList;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.VariableString;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.LiteralUtils;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Name("Raw String")
+@Description("Returns the string without formatting (colors etc.) and without stripping them from it, " +
+	"e.g. <code>raw \"&aHello There!\"</code> would output <code>&aHello There!</code>")
+@Examples("send raw \"&aThis text is unformatted!\" to all players")
+@Since("INSERT VERSION")
+public class ExprRawString extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprRawString.class, String.class, ExpressionType.COMBINED, "raw %strings%");
+	}
+
+	@SuppressWarnings("NotNullFieldNotInitialized")
+	private Expression<? extends String> expr;
+	@SuppressWarnings("NotNullFieldNotInitialized")
+	private Expression<? extends String>[] messages;
+
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		expr = LiteralUtils.defendExpression(exprs[0]);
+		messages = expr instanceof ExpressionList<?> ?
+			((ExpressionList<String>) expr).getExpressions() : new Expression[]{expr};
+		for (Expression<? extends String> message : messages) {
+			if (message instanceof ExprColoured) {
+				Skript.error("The 'colored' expression may not be used with a 'raw string' expression");
+				return false;
+			}
+		}
+		return LiteralUtils.canInitSafely(expr);
```

Comment:
```suggestion
		return true;
```
this isn't needed, only when the expr is `%object%` (or plural) in the pattern

Reply:
The expression _is_ plural

Reply:
I meant `%objects%`, its plural form, not any plural type

Reply:
ah, alright